alter.sql

```diff
@@ -0,0 +1,24 @@
+alter table animals
+add column species varchar;
+
+begin;
+update animals;
+set species = 'unspecified';
+select * from animals;
+rollback;
+select * from animals;
+
+begin;
+update animals 
+set species = 'digimon' where name ~ '.*(mon)';
+commit;
+
```

Comment:
- [ ] You miss this test requirement in your SQL command. 👇🏻 Kindly add it.

![tomor1](https://user-images.githubusercontent.com/79058364/165298649-3028ecb7-8b23-4fea-bbcf-68b701237640.png)

- Don't forget all these SQL commands must be written inside the `queries.sql` file

Reply:
Ok moved all to `queries.sql`

Reply:
Great 👍🏻